### PR TITLE
Add cookie consent endpoint and update copy

### PR DESCRIPTION
**Pull request by debugwand**
🐿 v2.8.0

### CODE REVIEW

File: src/components/cookie-consent/main.js [hunk 1]
```diff
@@ -16,24 +16,44 @@ module.exports = function customSetup (banner, done) {
...
+		const elemAction = elem.getAttribute('action');
+		const elemMethod = elem.getAttribute('method');
+		event.preventDefault();
+		// call the consent proxy to set other cookies to yes, but still hide banner if those fail. User can manage via preferences pages.
+		return fetch(elemAction, {
+			method: elemMethod
+		})
+		.then(setAccepted)
+		.catch(error => {
+			return { error };
```
Comment (jonnywyatt) [inline src/components/cookie-consent/main.js]: Will this get logged out anywhere?

File: src/components/cookie-consent/main.js [hunk 2]
```diff
@@ -16,24 +16,44 @@ module.exports = function customSetup (banner, done) {
...
+		});
+	};
+
 	const setup = () => {
 		const accepted = [].slice.call(wrapper.querySelectorAll('[data-action="accepted"]'));
-		accepted.forEach(elem => elem.addEventListener('click', (event) => accept(elem, event), false));
+		const acceptForm = [].slice.call(wrapper.querySelectorAll('[data-action="accept-form"]'));
```
Comment (jonnywyatt) [inline src/components/cookie-consent/main.js]: You can call forEach directly on the nodeList results - https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll

Comment (debugwand) [reply src/components/cookie-consent/main.js]: I initially did this and had to change it, trying to recall why. Will dig through commits

Comment (debugwand) [reply src/components/cookie-consent/main.js]: yep IE11: https://github.com/Financial-Times/n-messaging-client/commit/29ace9df5a4c7868c2341c98f54a568e3d777ed0

File: src/components/cookie-consent/main.js (hunk 1 above)
Comment (debugwand) [reply src/components/cookie-consent/main.js]: o-errors I think is picking this up

**Pull request merged**